doc/Options/zCOBOL_filetypes.md

```diff
@@ -0,0 +1,12 @@
+# zCOBOL file types
+
+TYPE | Format | File Description       | File or Report Format Description
+-----|--------|------------------------|---
+CBL  | ASCII  | COBOL source program   | 1-6 sequence #, 7 Comment if not space, 8-11 area A, 12-72 area B
+CPZ  | ASCII  | COBOL copy book member | 1-6 sequence #, 7 Comment if not space, 8-11 area A, 12-72 area B
+MLC  | ASCII  | Macro assembler source program generated by phase 1 of the zcobol compiler which uses zcobol.class regular expression based parser in z390.jar to read CBL source file and create MLC source file in one pass. | Macro call. for each COBOL statement starting in area A and for each COBOL verb found in area B.  Working storage data items are mapped to WS macro call with level as first parameter.  Each macro call name is followed by positional  parameters found following verb up to next verb or period.   Periods are mapped to PERIOD macro call.  Parameters of the form keyword(..) are passed as single parameter.  Other ( and ) are passed as separate parameter in quotes.
+BAL  | ASCII  | HLASM compatible source code generated by phase 2 of the zCOBOL compiler when using ZC390C or ZC390CLG commands.|HLASM compatible source statements generated by the zcobol+zcobol\z390 macros during expansion of the generated MLC file.
+JAVA | ASCII  | Java compatible source program file generated by phase 2 of the zCOBOL compiler when using ZCJAVC or ZCJAVCLG commands.|Java source statements generated by the zcobol+zcobol\java macros during expansion of the generated MLC file.
+CPP  | ASCII  | MS Visual Express C compatible source program file generated by phase 2 of the zCOBOL compiler when using ZCVCEC or ZCVCECLG commands.| MS Visual Express C compatible source statements generated by the zcobol+zcobol\vce macros during expansion of the generated MLC file.
+HLA  | ASCII  | HLA/MASM compatible source program file generated by phase 2 of the zCOBOL compiler when using ZC586C or ZC586CLG commands.|HLA/MASM compatible source statements generated by the zcobol+zcobol\i586 macros during expansion of the generated MLC file.
+CPY|ASCII|Generated copy file containing macro calls to define labels defined in a ZCOBOL program.| LABEL generated ZCOBOL name
```

Comment:
Line 12.
1. I do not see the CPY line in the document I found at http://z390.sourceforge.net/zcobol/zcobol_options.htm.
2. If you leave this entry in, add missing period at end.

Reply:
This was in a spreadsheet that was includes as part of the latest release of z390. @z390org - Are you able to confirm that the repo spreadsheet https://github.com/z390development/z390/blob/main/doc/z390_options.xlsx is the latest for the project and the z390 site is just a bit behind.

Reply:
Its also in the doco on the z390.info site.. see http://www.z390.info/doc/ZCOBOL%20Files%20Types.pdf

Reply:
Fixed aff28bd